module/solutions/helper.js

```diff
@@ -2404,6 +2413,65 @@ module.exports = class SolutionsHelper {
     });
   }
 
+   /**
+   * List Solutions using organization id.
+   * @method
+   * @name listOrganizationSolutions
+   * @query {String} type - Assets type (program/solutions).
+   * @returns {Object} - Details of the solution under the organization.
+   */
+
+   static listOrganizationSolutions(bodyData){
```

Comment:
@praveenKDass this name and description is not right. It is giving solutions not just from an org but specifically where the user is an author too. Also cannot the list function be used directly instead of one more new function.